test/functional/test_framework/test_framework.py

```diff
@@ -1071,6 +1071,41 @@ def activate_dip8(self, slow_mode=False):
                 self.sync_blocks()
         self.sync_blocks()
 
+    def advance_to_locked_in_softfork(self, name, expected_locked_in_height):
+        self.log.info("Wait for " + name + " locked_in phase")
```

Comment:
`f-string` IMO is more convenient for logs:
```suggestion
        self.log.info(f"Wait for {name} locked_in phase")
```